README.md

```diff
@@ -58,7 +58,7 @@ To install these userstyles:
 > **Note** <br>
```

Comment:
Are we intentionally disabling the GitHub callout syntax here or?

Reply:
What do you mean?

Reply:
![Screenshot 2023-10-08 at 13 53 37 (Arc)](https://github.com/catppuccin/userstyles/assets/47499684/18a2a1d4-4311-4920-b328-8908257aa888)

vs.

> [!NOTE]
> If you would like to submit a userstyle or learn how to port Catppuccin to a website, please refer to the [userstyle-creation.md](docs/userstyle-creation.md).

Reply:
What we have right now looks like the old GitHub callout syntax and I was guessing the `<br>` was there to disable it?

Reply:
This can be swapped then, this was just an oversight and not spotted before then

Reply:
Alright. Should I do that  -  switch all of the `> **Note** <br>`s to `> [!NOTE]`s (in `userstyles.yml` and `./README.md`) - in this PR?

Reply:
Yes go ahead